Wikipedia/UI-V5/WMFArticleListCollectionViewController.m

```diff
@@ -32,6 +32,10 @@ @interface WMFArticleListCollectionViewController ()
 @implementation WMFArticleListCollectionViewController
 @synthesize listTransition = _listTransition;
 
+- (instancetype)init {
```

Comment:
There hasn't yet been a need to instantiate a list VC programmatically.  The current setup seems to support both, since there's basically 0 configuration _of the list VC_ happening in the storyboard.